apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -4213,21 +4213,33 @@ static void tc_libc_math_ynf(void)
  */
 static void tc_libc_math_gamma(void)
 {
-	double value;
-	double result;
-
-	value = 2.0;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, 1);
-
-	value = 3.0;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, 2);
-
-	value = INFINITY;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, (double)INFINITY);
-
+	double input[13] = { 2.0, 3.0, INFINITY, 256.5,
```

Comment:
Like other testcases we use 'in_val' and 'sol_val' variable names instead of 'input' and 'compare' so can you please do that to maintain similarity.

Reply:
modify